README.md

```diff
@@ -882,22 +886,22 @@ Configures a deployed local pool (either Burn-Mint or Lock-Release) to recognize
 **Usage:**
 
 ```shell
-npx hardhat configure-pool --network <localNetworkName> 
---local-pool <localPoolAddress> 
---remote-network <remoteNetworkName> 
---remote-pool <remotePoolAddress> 
---remote-token <remoteTokenAddress> 
+npx hardhat configure-pool --network <localNetworkName>
+--local-pool <localPoolAddress>
```

Comment:
just saw that for sending tokens we use `source` and `destination` language but here we're using `local` and `remote`

should we not be consistent with public docs etc and stick to source/dest?